dags/common/cleanup.py

```diff
@@ -18,7 +20,37 @@ def convert_html_subsripts_to_latex(input):
 def clean_collaboration(input):
     return clean_whitespace_characters(input.replace("for the", ""))
 
+
 def remove_specific_tags(value, tags=None, attributes=None):
     tags = tags or []
     attributes = attributes or []
     return clean(value, tags=tags, attributes=attributes, strip=True)
+
+
+class RemoveUnneccessaryTagsFilter(Filter):
+    """Filter removes content of label, but not the tag itself!"""
+
+    def __iter__(self):
+        label_tag = False
+        for token in Filter.__iter__(self):
+            if token["type"] == "StartTag" and token["name"] == "label":
+                label_tag = True
+            elif token["type"] == "EndTag":
+                label_tag = False
+            elif label_tag:
+                token["data"] = ""
```

Comment:
```suggestion
            if token["type"] == "StartTag" and token["name"] == "label":
                 token["data"] = ""
```

Reply:
We cannot use it like this, because the output regarding the label of iteration is:
```
....
{'type': 'StartTag', 'name': 'label', 'namespace': None, 'data': {}}
{'data': 'Label', 'type': 'Characters'}
{'type': 'EndTag', 'name': 'label', 'namespace': None}
....
```
We need to change the value in 2nd line ;)

Reply:
Ok I see, thanks for the explanation, we should change it a bit just to make it more explicit:

```

 token["type"] == "EndTag" and token["name"] == "label":
     label_tag = False


     
```